README.md

```diff
@@ -74,6 +74,61 @@ perfplot log_Array1k_19-09-2019_12-13-49
 
 This will generate a file called `log_Array1k_19-09-2019_12-13-49.pdf`.
 
+# Save results to SQL database
+
+## Requirements
+***ODB 2.5.0 (this is required for gcc 7.4.0):***
+Follow the instructions [here](https://www.codesynthesis.com/products/odb/doc/install-build2.xhtml)
+ to install build2 toolchain, odb compiler and runtime libraries.
+
+Remember to install the correct gcc plugin, for example (for gcc 7):
+```
+sudo apt-get update
+sudo apt-get install gcc-7-plugin-dev
+```
+Additionally, if you want to use MYSQL database, you will need to build a not yet released version,
+so instead of executing:
+```
+bpkg build libodb-mysql
+bpkg install libodb-mysq
+
+```
+you will need to execute:
+```
+bpkg build libodb-mysql@https://git.codesynthesis.com/odb/libodb-mysql.git#fix-bind-decl
+bpkg install libodb-mysq
+```
+After installing all the libraries make sure to add `/usr/local/lib` to your PATH:
+```
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+```
+
+***ODB 2.4.0:***
+All the downloads can be found [here](https://www.codesynthesis.com/products/odb/download.xhtml).
+* Odb compiler ([Instructions](https://www.codesynthesis.com/products/odb/doc/install-unix.xhtml))
```

Comment:
Are we supporting this version of odb (2.4) ? I thought we are going with 2.5 right? If not supporting 2.4 any longer these can be removed.

Reply:
It doesn't work for us, after we changed to gcc7 and ubuntu 18.04 it started to break. However, odb 2.4.0 works fine (and installation is so much easier) on ubuntu 16.04 and gcc 5. I'll update readme so users are aware of the two options and why to select odb 2.4.0 or 2.5.0.

Reply:
Okay I think that sounds good.